setup.cfg

```diff
@@ -15,11 +15,11 @@ packages =
     ovirtlago
 data_files =
 # core lago
-    /etc/lago.d = etc/lago.d/*
-    /etc/sudoers.d/ = etc/sudo/lago
-    /etc/polkit-1/localauthority/50-local.d = etc/polkit/*
+    etc/lago.d = etc/lago.d/*
+    etc/sudoers.d/ = etc/sudo/lago
+    etc/polkit-1/localauthority/50-local.d = etc/polkit/*
 # ovirt lago plugin
-    /etc/firewalld/services = etc/firewalld/services/*
```

Comment:
That depends on [available permissions](http://docs.openstack.org/developer/pbr/#files).